Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Commands/RunCommand.cs

```diff
@@ -28,6 +29,12 @@ public override async Task<int> ExecuteAsync(CommandContext context, RunCommandS
     {
         try
         {
+            if (settings.PrintVersionInfo)
```

Comment:
can we add a unit test to https://github.com/aws/aws-lambda-dotnet/blob/8c1705f523c9d75db87bc326f5ee3d5ec6de39be/Tools/LambdaTestTool-v2/tests/Amazon.Lambda.TestTool.UnitTests/Commands/RunCommandTests.cs#L17? 

Also as a side note, do we want to enforce unit test coverage for future PRs as a check? What do we do for other projects that we have?

Reply:
I was being reluctant adding a mocking layer around stdout so I could reliable unit test this. Turns out we are already mocking stdout with `IToolInteractiveService` so I added the unit test.